docs/docs/guides/asynchronous-data-queries.md

```diff
@@ -179,12 +179,7 @@ const friendsInfoQuery = selector({
   key: 'FriendsInfoQuery',
   get: ({get}) => {
     const {friendList} = get(currentUserInfoQuery);
-    const friends = [];
-    for (const friendID of friendList) {
-      const friendInfo = get(userInfoQuery(friendID));
-      friends.push(friendInfo);
-    }
-    return friends;
+    return friendList.map(get(userInfoQuery);
```

Comment:
For explanatory purposes in documentation let’s really emphasize that `userInfoQuery()` is a function. Also, this isn’t valid because `get()` doesn’t return a function. 

```
return friendList.map(friendID => get (userInfoQuery(friendID)));
```

Reply:
Of course, I'm sorry I went to fast on this. It should be better now.